canonical/ubuntu.com
# New Ubuntu Pro Page

## Done

- New page for Ubuntu Pro

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Visit [https://ubuntu-com-12087.demos.haus/advantage](https://ubuntu-com-12087.demos.haus/advantage)

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5995

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
